docs/introduction/path_to_chrysalis.md

```diff
@@ -8,81 +8,77 @@ Progress in Coordicide research has resulted in identifying many concepts that c
 
-The second phase of Chrysalis consists of:
+The second phase of Chrysalis consisted of:
 
-- Reusable addresses and support for more standard cryptography (EdDSA), making efficient hardware support for all major architectures possible
-- A simplified transaction layout and a reduction in transaction size, further increasing performance and efficiency
-- Significant improvements to the usability and reliability of IOTA
-- A switch to a UTXO based model from the current account model
+- Reusable addresses and support for more standard cryptography (EdDSA), making efficient hardware support for all major architectures possible.
+- A simplified transaction layout and a reduction in transaction size, further increasing performance and efficiency.
+- Significant improvements to the usability and reliability of IOTA.
+- A switch to a UTXO based model from the previous account model.
 
-The introduction of reusable addresses is an important change for token holders. This will vastly improve IOTA’s usability and make integration into new exchanges, wallets, and payment systems much simpler. A new wallet, called Firefly, will be released with Chrysalis. This wallet will allow token holders to transition from the current WOTS address scheme to the new EdDSA scheme.  
+The introduction of reusable addresses was an important change for token holders. This vastly improved IOTA’s usability and made integration into new exchanges, wallets, and payment systems much simpler. A new wallet, called Firefly, was released with Chrysalis which allowed token holders to transition from the WOTS address scheme to the new EdDSA scheme.  
 
-Our goal is to make this transition as seamless as possible for everyone in the IOTA ecosystem. This includes a variety of improvements and updates to our libraries and software, as well as training and educational sessions for our partners.
+### From Plan to Action
 
-### From plan to action
+With Chrysalis, we had to make decisions on how to best implement all the upcoming changes, ensuring a correct and timely implementation. What follows is an overview of the various components we worked on to successfully implement this update.
 
-With Chrysalis, we had to make decisions on how to best implement all the upcoming changes, ensuring a correct and timely implementation. What follows is an overview of the various components we are working on to successfully implement this update.
+### Specification and Standardization
 
-### Specification and standardization
+Specifications are a major part of our new development process. All our new software projects (Node software, Wallet, Identity, Access, Streams, etc.) were based on vetted specifications. Specifications make it possible for external parties to follow a project’s intended functionality, for example for audit purposes, or develop their own implementations in different languages.
```

Comment:
- we should add links to the mentioned projects.
- I don't think we should etc. our projects.